styles/simple.scss

```diff
@@ -1,18 +1,106 @@
 .n-newsletter-signup--simple {
-	display: block;
+	display: flex;
+	flex-direction: row;
+	flex-shrink: 0;
 	border: 0;
+	gap: 20px;
+	padding-top: 20px;
+	background-color: oColorsByName("white-80");
+	padding-left: 24px;
+	padding-bottom: 0;
+
+	@include oGridRespondTo($from: M) {
+		align-items: start;
+		padding-left: 44px;
+		padding-bottom: 20px;
+	}
+
+	.n-newsletter-signup__card_content {
+		display: flex;
+		flex-direction: column;
+		align-items: start;
+		padding-left: 0;
+
+		@include oGridRespondTo($from: M) {
+			flex-direction: row;
+			align-items: center;
+		}
+
+	}
+
+	.n-newsletter-signup__body {
+		@include oTypographySans($scale: 0);
+		color: oColorsByName('black-80');
+		display: inline-block;
+		width: fit-content;
+
+		@include oGridRespondTo($from: M) {
+			width: 70%;
+		}
+	}
+
+	.n-newsletter-signup__top-meta {
+		top: 0;
+		right: 0;
+	}
+
+
+	.n-newsletter-signup__flex-child-left {
+		position: relative;
+		text-align: center;
+		color: oColorsByName('claret');
+		flex: 0 0 100%;
+		max-width: 158px;
+		max-height: 89px;
+
+		@include oGridRespondTo($from: M) {
+			max-width: 158px;
+			max-height: 89px;
+			flex: 0 0 30%;
+			overflow: hidden;
+		}
+
+		&__card_image {
+			max-width: 100%;
+		}
```

Comment:
Question: I also cannot see these styles being used anywhere in the demos so can you explain where they are used. If they are used we need a demo for them and if they are not then they can be removed


**Change Request:** Using `&` here is not consistent. It also introduces a couple of issues the first one being it will produce a style name that is not BEM as the style name will be `.n-newsletter-signup__flex-child-left__card_image`

Also I would encourage us to not use `&` because it makes the code harder to read and harder to search for styles.